x-pack/plugins/security_solution/common/endpoint/utils/commands.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type {
+  ConsoleResponseActionCommands,
+  ResponseActionsApiCommandNames,
+} from '../service/response_actions/constants';
+import { RESPONSE_CONSOLE_ACTION_COMMANDS_TO_REQUIRED_AUTHZ } from '../service/response_actions/constants';
+import type { EndpointPrivileges } from '../types';
+
+export const getRbacControl = ({
```

Comment:
This is a UI specific construct. `EndpointPrivileges` has some additional ui specific property (like `loading`). On the server side we use `EndpointAuthz`.

I would not recommend trying to make this reusable with the server side.